docs/index.rst

```diff
@@ -8,7 +8,8 @@ Qiskit Terra documentation
 
   API References <apidocs/terra>
   Release Notes <release_notes>
-
+  Migration Guides <migration_guides/index>
```

Comment:
FYI, qiskit.org/documentation is built by the meta package not terra. So in order to add migration guides to the sidebar, we need to make a corresponding PR in the meta package. I can create one. It is still necessary to this file like you did though, because this is useful for terra developers to build terra doc locally and for ci.

Reply:
@ElePT The draft PR is here: https://github.com/Qiskit/qiskit/pull/1664. It's on hold until at least one of the migration guides PR is merged.

Reply:
Thanks!! [HuangJunye](https://github.com/HuangJunye)